### ISSUE TITLE
Responsiveness of design on large screens (like 4k displays)

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/server#7192]
> ![bildschirmfoto von 2017-11-16 11-45-27](https://user-images.githubusercontent.com/213943/32887322-cc8686fe-cac3-11e7-8184-ce507f49f48e.png)

I have a 4k monitor and still the word "Profile picture" is truncated? The text is not even available as title...

Can we improve this somehow? 

cc @nextcloud/designers

**Comment by skjnldsv** [comment]
Adding this to *the list*! 😂

**Comment by jancborchardt** [comment]
@nickvergessen do you have two monitors side by side, or is the bottom truncated? Could you post a screenshot of the whole screen?

@skjnldsv @nextcloud/designers we need to talk about responsiveness also for huge displays. Like flexible units, automatic zooming, etc.

**Comment by nickvergessen** [comment]
> do you have two monitors side by side, or is the bottom truncated? Could you post a screenshot of the whole screen?

I cut it off, can post a full screen tomorrow

**Comment by skjnldsv** [comment]
@jancborchardt I will work on responsiveness when I''ll start removing the absolute mess (pun intended) of the css content! :p

**Comment by pixelipo** [comment]
But some issues must be decided first. Some items still must have fixed `max-width` otherwise it will look stupid

**Comment by nickvergessen** [comment]
Here is a full screen just for jan:
> ![bildschirmfoto von 2017-11-17 09-03-37](https://user-images.githubusercontent.com/213943/32937024-ffac0e1a-cb76-11e7-8a34-3f5b6d8892fa.png)

**Comment by nickvergessen** [comment]
Also I just notices: App management has no row hover effect, so it's hard to make the link between enable/disable button and the app name
> ![bildschirmfoto von 2017-11-17 09-06-14](https://user-images.githubusercontent.com/213943/32937101-467661a6-cb77-11e7-83eb-873c835f55b0.png)

**Comment by jancborchardt** [comment]
Ok, so we definitely should use flexible units so on a 4k monitor stuff should not be so small. @nickvergessen cause it is quite small, right? I assume the monitor is smth between 24"–30"?

@nickvergessen can you open a separate issue about the hover in user mgmt?

**Comment by nickvergessen** [comment]
The screen is 27" it's a bit small, but not really harder to read compared with pages like twitter, fb, gh,... they are all pretty much the same.

## File list without sidebar
> ![bildschirmfoto von 2017-11-17 09-20-50](https://user-images.githubusercontent.com/213943/32937500-1f552402-cb79-11e7-8c13-1656004ed55e.png)

## File list with sidebar
> ![bildschirmfoto von 2017-11-17 09-21-06](https://user-images.githubusercontent.com/213943/32937506-2795bde8-cb79-11e7-844d-e4a056381a04.png)

## User management
> ![bildschirmfoto von 2017-11-17 09-21-19](https://user-images.githubusercontent.com/213943/32937513-31c739d6-cb79-11e7-8044-5f8ce7557a2e.png)

**Comment by szaimen** [comment]
I'm closing this issue due to inactivity. Please reopen if you still want to implement this.
